schema.sql

```diff
@@ -136,7 +136,7 @@ CREATE TABLE IF NOT EXISTS `posts` (
   `updated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `view_count` int(6) NOT NULL,
   `title` varchar(160) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
-  `content` text CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
+  `content` mediumtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
```

Comment:
This change isn't needed -- `schema.sql` only represents the initial state of the database, and then all migrations are applied over it.